quiz.rb

```diff
@@ -2,19 +2,95 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
+
+class Dog
+  def initialize(name, breed, age)
+    @name = name
+    @breed = breed
+    @age = age
+  end
+
+  attr_reader :name, :breed, :age
+end
+
 # Question 2
 # Create a new Dog object.
+pig = Dog.new("pig", "pug", 11)
 
 # Question 3
 # Create a Person class, with attributes name, age, and location.
 # It should set all three attributes when you create a new Person.
 # Name and location should be writeable, but age should be read-only.
 
+class Person
+  def initialize(name, age, location)
+    @name = name
+    @age = age
+    @location = location
+  end
+
+  attr_reader :age
+  attr_accessor :name, :location
+
+end
+
 # Question 4
 # Create a new Person object using parameters "Dave", 32, and "Ohio".
 # Then, set that Person's location to "Somerville"
+Dave = Person.new("Dave", 32, "Ohio")
```

Comment:
Another convention comment - generally, only class names are capitalized. Should still work, though.